packages/d3fc-webgl/src/program/programBuilder.js

```diff
@@ -26,7 +26,16 @@ export default () => {
         );
         buffers(context, program, verticesPerElement, count);
 
-        context.drawArrays(mode, 0, count * verticesPerElement);
+        if (buffers.elementIndices() != null) {
+            context.drawElements(
+                mode,
+                buffers.elementIndices().data().length * count,
```

Comment:
`buffers.elementIndices().data().length` === `verticesPerElement`?

Reply:
No.

`verticesPerElement` = number of corners defined.
`elementIndices length` = number of vertices defined (via indices).

Reply:
Of course. I think we should now be able to remove `verticesPerElement`, although let's do that in another PR.